### ISSUE TITLE
Audio configuration?

### ISSUE THREAD

**Issue by brylie** [BHSPitMonkey/ninjam-js#7]
Is it possible to configure the input and output devices for this app? Is audio configuration done through the browser, for instance?

**Comment by BHSPitMonkey** [comment]
Yes, it's up to the browser (essentially Chrome/Chromium if you're using
the Electron package). Output all goes to your default, and Input channels
should appear and be listed separately in the UI depending on what your OS
allows.
